spec/models/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe User, type: :model do
+    describe 'DB table' do
+        it { is_expected.to have_db_column :name }
+        it { is_expected.to have_db_column :email }
+        it { is_expected.to have_db_column :encrypted_password }
+    end
+
+    describe 'Validations' do
+        it { is_expected.to validate_presence_of :name }
+        it { is_expected.to validate_length_of :name }
+        it { is_expected.to validate_presence_of :email }
+        it { is_expected.to validates_uniqueness_of :name }
+    end
+
+    describe 'Factory' do
+        it 'should have valid Factory' do
+            expect(create(:user)).to be_valid
+        end
+    end
+end
```

Comment:
be consistent with the indentation, use 2 spaces not 4 in this whole file, otherwise the formatting is good